functions/zoom-uploader.py

```diff
@@ -96,10 +99,10 @@ def created(self):
         return self.data['start_time']
 
     @property
-    def created_local(self, tz='US/Eastern'):
+    def created_local(self, tz=LOCAL_TIME_ZONE):
```

Comment:
I'm wondering if it would be better to have both the `created` and `created_local` properites return datetime objects vs strings. then do the string formatting where they get used.

Reply:
Good idea.